code/datums/gamemode/factions/bloodcult/bloodcult_runes.dm

```diff
@@ -411,13 +409,23 @@ var/list/uristrune_cache = list()//icon cache, so the whole blending process is
 			to_chat(user, "<span class='danger'>You find yourself unable to focus your mind on the arcane words of the rune.</span>")
 			return
 
-	if(user.is_wearing_item(/obj/item/clothing/mask/muzzle, slot_wear_mask))
-		to_chat(user, "<span class='danger'>You are unable to speak the words of the rune because of \the [user.wear_mask].</span>")//TODO; SILENT CASTING ALLOWS MUZZLED CAST
-		return
+	var/silent_casting = 0
+	if (user.mind && user.mind.GetRole(CULTIST))
+		var/datum/role/cultist/C = user.mind.GetRole(CULTIST)
+		for (var/T in C.tattoos)
```

Comment:
Can't this be replaced by something of the form, `if ("Silent Casting" in C.tatoos)` (assuming you're never changing the names of the tattoos yourself